jupyterhub/notebook-images/overlays/additional/elyra-notebook-imagestream.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/thoth-station/s2i-lab-elyra"
+    opendatahub.io/notebook-image-name: "Elyra Notebook Image"
+    opendatahub.io/notebook-image-desc: "Jupyter notebook image with Elyra-AI installed"
+  name: s2i-lab-elyra
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - annotations:
+      openshift.io/imported-from: quay.io/thoth-station/s2i-lab-elyra
+    from:
+      kind: DockerImage
+      name: quay.io/thoth-station/s2i-lab-elyra:0.0.1
```

Comment:
The tag is `v0.0.1` (see https://quay.io/repository/thoth-station/s2i-lab-elyra?tag=v0.0.1&tab=tags)